.gitignore

```diff
@@ -1,8 +1,8 @@
+.DS_Store
```

Comment:
`.DS_Store` is a very local thing for mac users. I suggest you to add it to your own `.gitignore` (the one applying for every single project on your laptop)